### PR TITLE
PIP 12 cards

### PR DESCRIPTION
**Pull request by Simisays**
@paulsnoops what you think about the Junk token name ?

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/dogmeat_ever_loyal.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Name:Dogmeat, Ever Loyal
+ManaCost:R G W
+Types:Legendary Creature Dog
+PT:3/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigMill | TriggerDescription$ When NICKNAME enters the battlefield, mill five cards, then return an Aura or Equipment card from your graveyard to your hand.
+SVar:TrigMill:DB$ Mill | NumCards$ 3 | Defined$ You | SubAbility$ DBChangeZone
+SVar:DBChangeZone:DB$ ChangeZone | Origin$ Graveyard | Destination$ Hand | ChangeType$ Aura.YouOwn,Equipment.YouOwn | Hidden$ True | Optional$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/dogmeat_ever_loyal.txt:7]: why Optional?

File: forge-gui/res/cardsfolder/upcoming/radstorm.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Name:Radstorm
+ManaCost:3 U
+Types:Instant
+K:Storm
+A:SP$ Proliferate | SpellDescription$ Proliferate. (Choose any number of permanents and/or players, then give each another counter of each kind already there.)
+DeckNeeds:Type$Counters
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/radstorm.txt:6]: bad type

File: forge-gui/res/cardsfolder/upcoming/vault_101_birthday_party.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+SVar:DBChangeZone:DB$ ChangeZone | Origin$ Hand,Graveyard | Destination$ Battlefield | ChangeType$ Aura,Equipment | RememberChanged$ True |  ChangeNum$ 1 | SubAbility$ You may put an Aura or Equipment card from your hand or graveyard onto the battlefield. If an Equipment is put onto the battlefield this way, you may attach it to a creature you control.
+SVar:DBAttach:DB$ Attach | Optional$ True | ChoiceTitle$ You may choose a creature you control to attach that equipment to | ConditionDefined$ Remembered | ConditionPresent$ Equipment | Object$ Remembered | Choices$ Creature.YouCtrl | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+DeckNeeds:Type$Aura|Equipment
+DeckHas:Ability$Token & Type$Soldier|Food
+Oracle:(As this Saga enters and after your draw step, add a lore counter. Sacrifice after III.)\nI — Create a 1/1 white Human Soldier creature token and a Food token. (A Food token is an artifact with "{2}, {T}, Sacrifice this artifact: You gain 3 life.")
+II, III — You may put an Aura or Equipment card from your hand or graveyard onto the battlefield. If an Equipment is put onto the battlefield this way, you may attach it to a creature you control.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/vault_101_birthday_party.txt:12]: bad break

**Comment by paulsnoops** [comment]
> @paulsnoops what you think about the Junk token name ?

Looks good to me!

File: forge-gui/res/cardsfolder/upcoming/intelligence_bobblehead.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
+Name:Intelligence Bobblehead
+ManaCost:3
+Types:Artifact Bobblehead
+A:AB$ Mana | Cost$ T | Produced$ Any | SpellDescription$ Add one mana of any color.
+A:AB$ Draw | Cost$ 5 T | NumCards$ X | SpellDescription$ Draw X cards, where X is the number of Bobbleheads you control.
+SVar:X:Count$Valid Bobblehead.YouCtrl
+DeckNeeds:Type$Bobbleheads
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/intelligence_bobblehead.txt:7]: bad type

Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/intelligence_bobblehead.txt:7]: bad type

File: forge-gui/res/cardsfolder/upcoming/nuka_cola_vending_machine.txt [hunk 5]
```diff
@@ -0,0 +1,9 @@
+Name:Nuka-Cola Vending Machine
+ManaCost:3
+Types:Artifact
+A:AB$ Token | Cost$ 1 T | TokenAmount$ 1 | TokenScript$ c_a_food_sac | TokenOwner$ You | SpellDescription$ Create a Food token. (It's an artifact with "{2}, {T}, Sacrifice this artifact: You gain 3 life.")
+T:Mode$ Sacrificed | ValidCard$ Food.YouCtrl | Execute$ TrigToken | TriggerZones$ Battlefield | TriggerDescription$ Whenever you sacrifice a Food, create a tapped Treasure token. (It's an artifact with "{T}, Sacrifice this artifact: Add one mana of any color.")
+SVar:TrigToken:DB$ Token | TokenTapped$ True | TokenScript$ c_a_treasure_sac
+DeckHas:Ability$Token & Type$Treasure|Food
+DeckHas:Type$Food
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/nuka_cola_vending_machine.txt:8]: bad line

File: forge-gui/res/lists/TypeLists.txt [hunk 6]
```diff
@@ -325,6 +325,7 @@ Shrine
 Shard
 [ArtifactTypes]
 Blood
+Bottlehead:Bottleheads
```
Comment (tool4ever) [inline forge-gui/res/lists/TypeLists.txt:328]: more typos?

File: forge-gui/res/cardsfolder/upcoming/rex_cyber_hound.txt [hunk 7]
```diff
@@ -0,0 +1,12 @@
+Name:Rex, Cyber-Hound
+ManaCost:1 W U
+Types:Legendary Artifact Creature Robot Dog
+PT:2/2
+T:Mode$ DamageDone | CombatDamage$ True | ValidSource$ Card.Self | ValidTarget$ Player | TriggerZones$ Battlefield | Execute$ TrigMill | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, they mill two cards and you get {E}{E} (two energy counters).
+SVar:TrigMill:DB$ Mill | Defined$ TriggeredTarget | NumCards$ 2 | SubAbility$ DBEnergy
+SVar:DBEnergy:DB$ PutCounter | Defined$ You | CounterType$ ENERGY | CounterNum$ 2
+A:AB$ ChangeZone | Cost$ PayEnergy<2> | SorcerySpeed$ True | Origin$ Graveyard | Destination$ Exile | ValidTgts$ Creature | WithCountersType$ BRAIN | AILogic$ Mairsil | SpellDescription$ Choose target creature card in a graveyard. Exile it with a brain counter on it. Activate only as a sorcery.
+S:Mode$ Continuous | Affected$ Card.Self | EffectZone$ Battlefield | GainsAbilitiesOf$ Card.YouOwn+counters_GE1_BRAIN | GainsAbilitiesOfZones$ Exile | GainsAbilitiesLimitPerTurn$ 1 | Description$ NICKNAME has all activated abilities of all cards in exile with brain counters on them.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/rex_cyber_hound.txt:9]: wrong valid

File: forge-gui/res/cardsfolder/upcoming/dogmeat_ever_loyal.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/dogmeat_ever_loyal.txt]: still wrong default

File: forge-gui/res/cardsfolder/upcoming/overencumbered.txt [hunk 8]
```diff
@@ -0,0 +1,13 @@
+Name:Overencumbered
+ManaCost:1 W
+Types:Enchantment Aura
+K:Enchant opponent
+A:SP$ Attach | ValidTgts$ Opponent | AILogic$ Curse
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME enters the battlefield, enchanted opponent creates a Clue token, a Food token, and a Junk token.
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ c_a_clue_draw,c_a_food_sac,c_a_junk_sac_exileplay | TokenOwner$ EnchantedPlayer 
+T:Mode$ Phase | Phase$ BeginCombat | ValidPlayer$ Opponent.EnchantedBy | TriggerZones$ Battlefield | Execute$ TrigEffect | TriggerDescription$ At the beginning of combat on enchanted opponent’s turn, that player may pay {1} for each artifact they control. If they don't, creatures can't attack this combat.
+SVar:TrigEffect:DB$ Effect | UnlessCost$ X | UnlessPayer$ EnchantedPlayer | UnlessSwitched$ True | StaticAbilities$ StaticCantAttack
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/overencumbered.txt:9]: Duration missing

Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/overencumbered.txt:9]: why Switched?

File: forge-gui/res/cardsfolder/upcoming/dogmeat_ever_loyal.txt [hunk 9]
```diff
@@ -0,0 +1,12 @@
+Name:Dogmeat, Ever Loyal
+ManaCost:R G W
+Types:Legendary Creature Dog
+PT:3/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigMill | TriggerDescription$ When NICKNAME enters the battlefield, mill five cards, then return an Aura or Equipment card from your graveyard to your hand.
+SVar:TrigMill:DB$ Mill | NumCards$ 3 | Defined$ You | SubAbility$ DBChangeZone
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/dogmeat_ever_loyal.txt:6]: wrong Num

**Review by tool4ever — APPROVED** [review]

**Pull request merged**